meilisearch-release-changelog/v1.3.0.md

```diff
@@ -289,6 +289,7 @@ Done by @Kerollmops in (#3889)
 # Fixes 🐞
 * Fix matching results returned by the `exactness` ranking rules when there are hard separators present (#3824) @dureuill 
 * Fixes an issue where searching for words without `.` doesn't match any string in the index ([#151](https://github.com/meilisearch/product/discussions/151), [#3778](https://github.com/meilisearch/meilisearch/discussions/3778))
+* Fix case-sensitive search result issues with camelCasing.  e.g. a search for `dellonghi` would previously not return a document with `DeLonghi` in it. Note: this deactivates camel case segmentation that was introduced in v1.2.0 (#3921) @ManyTheFish
```

Comment:
```suggestion
* Fix case-sensitive search result issues with camelCasing.  For example a search for `delonghi` would previously not return a document containing `DeLonghi`. This deactivates camel case segmentation that was introduced in v1.2.0 (#3921) @ManyTheFish
```

Reply:
Hello @guimachiavelli, the typo made on `dellonghi` is expected ☺️

Reply:
I'll update the other changes and keep the typo, thanks Many!

Reply:
Ah, ok 😅 

Is this issue caused by the interaction between camel casing and typo tolerance?